cmd/kwil-admin/nodecfg/toml.go

```diff
@@ -149,6 +149,22 @@ hostname = "{{ .AppCfg.Hostname }}"
 {{- end }}
 {{- end }}
 
+#######################################################################
+###                     Snapshots Configuration                     ###
+#######################################################################
+[app.snapshots]
+
+# Enables snapshots
+enabled = {{.AppCfg.Snapshots.Enabled}}
+
+# Path to the snapshots directory
+snapshot_dir = "{{.AppCfg.Snapshots.SnapshotDir}}"
+
+# Heights at which the database is snapshotted.
+snapshot_heights = {{.AppCfg.Snapshots.RecurringHeight}}
```

Comment:
Same comment about more clear doc.  The template var *Recurring* provides the clearest clue that this is a modulo.